tests/fractional_scale_v1.cpp

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright © 2020 Canonical Ltd.
```

Comment:
We're not putting dates on our copyright. Also, are you a time traveller? :wink: